flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/StandaloneResourceManager.java

```diff
@@ -112,4 +103,21 @@ protected ResourceID workerStarted(ResourceID resourceID) {
 		return resourceID;
 	}
 
+	@Override
+	protected void setFailUnfulfillableRequest(boolean failUnfulfillableRequest) {
```

Comment:
Let's not override the method but introduce another method which says that you start/stop the timeout. For the non standalone implementations the method should do nothing.